sbc_compassion/models/sponsorship_correspondence.py

```diff
@@ -179,6 +183,21 @@ def _compute_states(self):
             else:
                 letter.b2s_state = letter.state
 
+    @api.multi
+    @api.depends('sponsorship_id')
+    def _compute_is_first(self):
+        """ Sets the value true if is the first letter. """
+        for letter in self:
+            if letter.sponsorship_id:
+                count = self.search_count([
+                    ('sponsorship_id', '=', letter.sponsorship_id.id),
+                    ('direction', '=', "Beneficiary To Supporter")
+                    ])
+                if count == 0:
+                    letter.is_first_letter = True
+                    break
```

Comment:
I would rather remove this `break` and remove line 199. By default, boolean fields are always set to False. This makes sure you will treat all the letters contained in `self`. With the break statement, you could skip some letters.